util/codec/codec.go

```diff
@@ -510,12 +500,7 @@ func peekBytes(b []byte, reverse bool) (int, error) {
 		// The byte slice is encoded into many groups.
 		// For each group, there are 8 bytes for data and 1 byte for marker.
 		marker := b[offset+encGroupSize]
-		var padCount byte
-		if reverse {
```

Comment:
`reverse` is always false, so I remove the other branch to improve test coverage